book/website/collaboration/ambassador-schemes/ambassador-schemes-personal-story.md

```diff
@@ -0,0 +1,145 @@
+
+From this discussion, we realised we actually had the capacity to run a scheme where early career researchers could travel to a conference but also engage further with the partnership and run their own community activities.
+It was also an ideal time to set it up (around two years into the five year partnership), as it meant we had research projects up and running that participants could integrate into their community activities; I think we were not established enough to run the scheme any earlier. 
+The principal benefit of the scheme for us was increasing the reach and engagement of the partnership by working with students in relevant research areas, connecting with their networks and overall bolstering our community. 
+
+At that time, if you wanted to engage with the partnership as an external researcher the main ways you could do this were through our open research funding calls or through existing community activities, such as attending our virtual seminars. 
+Therefore, in addition, we also felt this scheme would support me as the Research Community Manager to add ways for the growing external community to participate through the new community activities. 
+
+Lastly, the scheme would be a great opportunity for us as an academic-industry collaboration to play a role in supporting the career development of early career researchers and giving them experience of organisations from two different sectors (academia and industry), which we felt passionate about. 
+This type of cross-sectorial experience is hard to come by for early career researchers and a particular area in need of upskilling, highlighted in various UK government reports, such as the [UK's National Data Strategy](https://www.gov.uk/government/publications/uk-national-data-strategy/national-data-strategy).
+This also would deliver more tangible impact and legacy for the partnership beyond the research collaboration itself. 
+
+
+## Scheme Set-up
+
+The Turing Institute had a student scheme in place (the Enrichment Scheme) but this worked quite differently to what we wanted to do for the scholar scheme, so the set up was as a brand new programme, which with documentation and processes, was a huge undertaking. 
+As with any new project or activity, it was useful to think about what we wanted to achieve with the scheme and work backwards from there to brainstorm how it should work in practice. 
+
+We put together an initial scoping document, ensuring objectives of the scheme were in place and the roles and responsibilities supporting its implementation were well defined. 
+We took this to the partnership leadership team to get some early feedback. 
+The decision was taken to ensure that the activities would be community based, rather than research based due to intellectual property agreements in place between the Turing and Roche.
```

Comment:
```suggestion
The decision was taken to ensure that the activities would be community based, rather than research, based due to intellectual property agreements in place between the Turing and Roche.
```